config.go

```diff
@@ -701,11 +704,13 @@ func loadConfig() (*config, error) {
 	// For each of the RPC listeners (REST+gRPC), we'll ensure that users
 	// have specified a safe combo for authentication. If not, we'll bail
 	// out with an error.
-	err := enforceSafeAuthentication(cfg.RPCListeners, !cfg.NoMacaroons)
+	err := lncfg.EnforceSafeAuthentication(cfg.RPCListeners,
```

Comment:
similar comment as below wrt. line wrapping on the calls to `EnforceSafeAuthentcation` and `NormalizeAddresses` in this section :)